### PR TITLE
[Documentation] Update Functional Test documents to follow SDK approach

### PR DESCRIPTION
**Pull request by Batta32**
### Purpose
*What is the context of this pull request? Why is it being done?*

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

**Review by VictorGrycuk — APPROVED** [review]
LGTM

File: samples/csharp/assistants/virtual-assistant/VirtualAssistantSample.FunctionalTests/README.md [hunk 1]
```diff
@@ -1,90 +1,66 @@
...
+- [Scenarios](#scenarios)
+    - [1. Virtual Assistant greets a new user](#1-virtual-assistant-greets-a-new-user)
+    - [2. QnA model provides a response to a user](#2-qna-model-provides-a-response-to-a-user)
+- [Reference](#reference)
+    - [Variables](#variables)
+- [Glossary](#glossary)
+
+## Scenarios
+
+This sections described the testing scenarios for Virtual Assistants, for each one of them we provide a high level goal description of the primary test case.
```
Comment (fran893) [inline samples/csharp/assistants/virtual-assistant/VirtualAssistantSample.FunctionalTests/README.md:28]: This -> These

File: samples/csharp/skill/SkillSample.FunctionalTests/README.md [hunk 2]
```diff
@@ -1,60 +1,56 @@
...
+
+- [Scenarios](#scenarios)
+    - [1. Skill greets a new user](#1-skill-greets-a-new-user)
+- [Reference](#reference)
+    - [Variables](#variables)
+- [Glossary](#glossary)
+
+## Scenarios
+
+This sections described the testing scenarios for Skills, for each one of them we provide a high level goal description of the primary test case.
```
Comment (fran893) [inline samples/csharp/skill/SkillSample.FunctionalTests/README.md:27]: This -> These

File: templates/typescript/samples/sample-assistant/functional-tests/README.md [hunk 3]
```diff
@@ -1,104 +1,61 @@
...
+
+- [Scenarios](#scenarios)
+    - [1. Virtual Assistant greets a new user](#1-virtual-assistant-greets-a-new-user)
+- [Reference](#reference)
+    - [Variables](#variables)
+- [Glossary](#glossary)
+
+## Scenarios
+
+This sections described the testing scenarios for Virtual Assistants, for each one of them we provide a high level goal description of the primary test case.
```
Comment (fran893) [inline templates/typescript/samples/sample-assistant/functional-tests/README.md:27]: This -> These

File: templates/typescript/samples/sample-skill/functional-tests/README.md [hunk 4]
```diff
@@ -1,66 +1,56 @@
...
+
+- [Scenarios](#scenarios)
+    - [1. Skill greets a new user](#1-skill-greets-a-new-user)
+- [Reference](#reference)
+    - [Variables](#variables)
+- [Glossary](#glossary)
+
+## Scenarios
+
+This sections described the testing scenarios for Skills, for each one of them we provide a high level goal description of the primary test case.
```
Comment (fran893) [inline templates/typescript/samples/sample-skill/functional-tests/README.md:27]: This -> These

**Review by fran893 — CHANGES_REQUESTED** [review]

**Review by matiasroldan6 — APPROVED** [review]

**Pull request merged**